cmd/createtree/main_test.go

```diff
@@ -46,7 +58,7 @@ func TestRun(t *testing.T) {
 		HashStrategy:       trillian.HashStrategy_RFC_6962,
 		HashAlgorithm:      sigpb.DigitallySigned_SHA256,
 		SignatureAlgorithm: sigpb.DigitallySigned_RSA,
-		PrivateKey:         anyKey,
+		PrivateKey:         marshalAny(&keyspb.PrivateKey{Der: pemDer}),
```

Comment:
nit: It might be nicer to still do the marshaling at the start of the test, so you can report an error using `t.Fatalf()`.